### PR TITLE
Assorted UI Improvements

### LINKED ISSUE #4805: ui: node count should use correct pluralization
A 1-node cluster, while a degenerate case, should be shown as "1 Node." See:

https://www.dropbox.com/s/7itr2k74ioi3ned/Screenshot%202016-03-02%2022.07.29.png?dl=0

### LINKED ISSUE #4806: ui: remove Users page
I'm not sure how, but it already looks spartan, and it'll be even more so once we remove the `hashedPassword` field from the UI.

### LINKED ISSUE #4937: ui: rename Overview to Cluster
As discussed.

### LINKED ISSUE #4939: ui: Cluster section should have tabs
These tabs are:

1. Overview
2. Graphs (aggregated versions of node graphs)
3. Events

### PR DESCRIPTION
**Pull request by mrtracy**
This commit contains four small UI fixes, closing four issues.
- Remove User Page
- Renames "Overview" navigation option to "Cluster"
- Correctly pluralizes "Nodes" in cluster overview page; if there is only 1
  node, it now says "Node".
- The cluster page has been converted to use a Tabbed layout, with two tabs:
  Overview and Events. No additional content has been added at this time.

Fixes #4805
Fixes #4806
Fixes #4937
Fixes #4939 

screen shot 2016-03-09 at 11 31 49 pm

---

This change is [Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/5057)

### CODE REVIEW

File: ui/ts/pages/cluster.ts [hunk 1]
```diff
@@ -206,6 +235,8 @@ module AdminViews {
             } else {
               let allStats: Models.Proto.NodeStatus[] = nodeStatuses.allStatuses();
               let totalStats: Models.Proto.Status = nodeStatuses.totalStatus();
+
+              axis.title = axis.titleFn(allStats);
```
Comment (maxlang) [inline ui/ts/pages/cluster.ts]: You should probably copy this into the (largely duplicated) RenderGraphs function above. Also would this mean that, for the number visualizations, the title has to be defined as a function?

**Comment by maxlang** [comment]
Minor nit, but LGTM

File: ui/ts/pages/cluster.ts (hunk 1 above)
Comment (mrtracy) [inline ui/ts/pages/cluster.ts]: In any case, copied to RenderGraphs (even though no visualizations are currently rendered that way).

**Comment by mrtracy** [comment]
Merging this, CircleCI failing with known flakes only.

**Comment by mrtracy** [comment]
Nevermind, can't merge from phone be a use circleci failed. :/

**Pull request merged**